dlrover/python/elastic_agent/diagnosis/diagnosis_agent.py

```diff
@@ -79,13 +79,22 @@ def diagnose_training_failure(self, worker_context: WorkerContext) -> str:
         if worker_context.remaining_failovers > 0 and not failure_node:
             logger.info(
                 f"[{worker_context.worker_spec.role}] Worker group "
-                f"{worker_context.run_result.state.name}. "
+                f"{worker_context.run_result.state.name}, "
+                f"is failure node: {failure_node},"
                 f"{worker_context.remaining_failovers}/"
-                f"{worker_context.worker_spec.max_restarts}"
-                f" attempts left; will restart worker group"
+                f"{worker_context.worker_spec.max_restarts} "
+                f"attempts left; will restart worker group."
```

Comment:
just "attempts left; will restart worker group."

Reply:
leave a blank is better